integreatly/integreatly-1.3.0/integreatly-operator.1.3.0.clusterserviceversion.yaml

```diff
@@ -177,20 +180,23 @@ spec:
                 - create
                 - watch
                 - get
+            #Allow us to create oauth clients (Note could be limited further by using named resources)
             - apiGroups:
                 - oauth.openshift.io
               resources:
                 - oauthclients
               verbs:
                 - create
                 - get
+            #Allow us to get and update configmaps (Note seems overly permissive need to revisit)                
             - apiGroups:
                 - ''
               resources:
                 - configmaps
               verbs:
                 - get
                 - update
+            # Not sure why we need this one need to revisit    
```

Comment:
This is to rollout/instantiate deployment configs as part of the 3scale installation